fetcher/dockerhub.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+#
+# ** License **
+#
+# Home: http://resin.io
+#
+# Author: Andrei Gherzan <andrei@resin.io>
+#
+
+import os
+import logging
+from docker import Client
+from fetcher.tar import tarFetcher
+from modules.util import *
+
+log = logging.getLogger(__name__)
+
+class dockerhubFetcher(tarFetcher):
+
+    def __init__ (self, conffile, version, remote):
+        super().__init__()
+        self.remotefile = os.path.join(self.remote + ":resinhup-" + machine + "-" + version)
```

Comment:
As commented here https://github.com/resin-os/resin-yocto-scripts/pull/33#r79108514 , let's change the tag to be without `resinhup-`